rust/prql/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "_ch_rust_prql"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+prql-compiler = "0.8.1"
+serde_json = "1.0"
+
+[lib]
+crate-type = ["staticlib"]
+
+[profile.release]
+debug = true
+
+[profile.release-thinlto]
+inherits = "release"
+lto = "thin"
```

Comment:
I went to `thin` because build type (`release-thinlto`) suggest that and I haven't measured how does it impact the performance. And I haven't measured it, because I didn't feel the need for it.

Reply:
It uses `thinlto` in profile name because ClickHouse uses ThinLTO. And I think that it will be better to use here full LTO as in other places, even though I don't think that there will be any difference in performance.